roles/openshift_metrics/tasks/main.yaml

```diff
@@ -1,13 +1,14 @@
 ---
-- local_action: shell python -c 'import passlib' 2>/dev/null || echo not installed
+- name: Check that python-passlib is available on the control host
+  local_action: htpasswd path=/dev/null name=dummy state=present password=dummy
```

Comment:
This is dangerous, any malicious binary can be named as `htpasswd` (e.g. in `/usr/local/bin`)

Reply:
Uhmm.... the Ansible module `htpasswd` depends on passlib. You may confused with the command module w/ htpasswd as command.

See also: https://github.com/ansible/ansible/blob/7c4b91844da30faaa38881d3e4a7fd8e755b1c5d/lib/ansible/modules/web_infrastructure/htpasswd.py

Reply:
Its the ansible module which is able to generate htpasswds. I don't see it being used anywhere in openshift-ansible

`openshift-metrics` role also verifies that htpasswd binary is present (in a secure way, not running it):
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_metrics/tasks/install_support.yaml#L3

However it doesn't look like `python-passlib` is being used anywhere - would you like to file a PR to remove it?